readme.txt

```diff
@@ -177,8 +178,45 @@ Join us in further translating the Pods interface at: [https://translate.wordpre
 
 We also have a dedicated [Slack Chat](https://pods.io/chat/) channel to help our translators get started and to support them on the process.
 
+== Upgrade Notice ==
+
+= 2.7 =
+This upgrade requires a minimum PHP version of 5.3+ and WordPress 4.4+.
+
 == Changelog ==
 
+= 2.7 - December 4th 2017 =
+## New PHP & WP Minimum Version Requirements:
+- PHP Version Change Requirement 5.3+
+- WP Version Change Required 4.4+
+
+## Major Changes in 2.7 Release: Flexible Relationships:
```

Comment:
Maybe add link to https://www.youtube.com/watch?v=1h_5tQ6LmoY as it's a really well done Video?

Reply:
Done below